tools/build/bazel_status.sh

```diff
@@ -6,6 +6,12 @@ set -o pipefail
 
 echo STABLE_GIT_COMMIT $(git rev-parse HEAD)
 
-echo STABLE_VERSION $(cat ./VERSION)
+BRANCH=$(git branch --show-current)
+
+if [[ $BRANCH == "develop" ]]; then
+  echo STABLE_VERSION "develop"
```

Comment:
I like it. Just a note that this var is also baked into the CLI. So `airy version` built from develop will show `develop` but not a problem at all. Except.. it could be that this test fails `infrastructure/cli/integration/version_test.go`